packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1786,6 +1868,9 @@ class SemanticsNode extends AbstractNode with DiagnosticableTreeMixin {
       scrollExtentMax: data.scrollExtentMax != null ? data.scrollExtentMax : double.nan,
       scrollExtentMin: data.scrollExtentMin != null ? data.scrollExtentMin : double.nan,
       transform: data.transform?.storage ?? _kIdentityTransform,
+// TODO(goderbauer): enable when engine is updated.
```

Comment:
Waiting for review and roll of https://github.com/flutter/engine/pull/7282 into this PR.